prometheus_rules/recording.rules.yml

```diff
@@ -0,0 +1,25 @@
+- name: gluster_volume
```

Comment:
aggregation rules look fine to me and as I understand all these aggregations are at volume level.